### ISSUE TITLE
Influx 0.8 DELETE - where should I execute query mentioned in the documentation

### ISSUE THREAD

**Issue by sirkubax** [influxdata/influxdb#7103]
I know that we have a new version, and I'm migrating towards, but I do have some old 0.8.8 version running.

What is the delete command in that version?
#curl -G 'http://influx01:8086/db/latency/series?u=user&p=pass' --data-urlencode "q=delete from logstash where time < now() -30d" -v
< HTTP/1.1 403 Forbidden

via web 8083 interface ExploreData window:
Insufficient permissions to write to latency

I'm not sure what are the API endpoints for the curl -X DELETE 

Should I run a GET curl with delete as mentioned in the documentation:
**[https://docs.influxdata.com/influxdb/v0.8/api/query_language/#deleting-data-or-dropping-series](https://docs.influxdata.com/influxdb/v0.8/api/query_language/#deleting-data-or-dropping-series)**

Should I Use the web interface?

What would be the place **from where (how) would You execute delete commands** mentioned in the documentation (curl command, interface, etc)

**Comment by sirkubax** [comment]
Looking at the https://github.com/influxdata/influxdb/blob/v0.8.8/api/http/api.go
I guess there delete should be executed via Query GET command?

**Comment by BrannonKing** [comment]
This question should be moved to the InfluxDB Google Group.

**Comment by sirkubax** [comment]
Yes and No

This is software or documentation issue!

**Comment by jsternberg** [comment]
Hello @sirkubax. @BrannonKing is right. This is a usability question and should be sent to the InfluxDB [mailing list](https://groups.google.com/forum/#!forum/influxdb). If the conversation from that results in a need for further documentation, an issue can be filed after a resolution is found. This issue tracker is only for bugs and feature requests.

Thanks!

**Comment by sirkubax** [comment]
Partialy solved here

https://groups.google.com/forum/#!msg/influxdb/rvOWOskOUfk/tpPluT05BgAJ

TL;dr - permission problem
new user with admin:true worked!
